core/storage/statistics/gae_models.py

```diff
@@ -875,6 +875,24 @@ def save_multi(cls, exploration_stats_dicts):
             exploration_stats_models.append(stats_instance)
         cls.put_multi(exploration_stats_models)
 
+    def to_dict(self):
```

Comment:
I don't think you need this at all -- please delete.

- to_dict() and other stuff should be defined on domain objects, not storage model instances (which should have as little logic as possible -- the basic idea is that we convert the storage models to domain objects as soon as we retrieve them from the datastore, and work exclusively with the plain domain objects thereafter).
- You're using to_dict() above on the domain objects anyway, so you don't need this one.

Reply:
Done.